src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js

```diff
@@ -13,8 +14,9 @@ $(document).ready(function () {
     ];
 
     dropDowns.forEach(function (dd) {
-      me.updateStateForDropDown(dd);
+        me.updateStateForDropDown(dd);      
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)